client/src/components/Flights/FlightTile.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    width: '100%',
+  },
+  paper: {
+    padding: theme.spacing(2),
+    margin: 'auto',
+    width: '100%',
+  },
+  image: {
+    width: 128,
+    height: 128,
+  },
+  img: {
+    margin: 'auto',
+    display: 'block',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+}));
+
+const displaySegments = (itinerary, arriveDepart) => {
+  if (itinerary.segments.length === 1) {
+    let arrivalTime = itinerary.segments[0][arriveDepart].at;
+    arrivalTime = arrivalTime.split('T');
+    const date = arrivalTime[0];
+    const time = arrivalTime[1];
+    return `${itinerary.segments[0][arriveDepart].iataCode} @
+    ${time} ${date}`;
```

Comment:
Should cast to date and extract time and date that way. Using this method will account for timezone information